src/aws/osml/model_runner/app.py

```diff
@@ -374,28 +369,24 @@ def queue_region_request(
         :param raster_dataset: Dataset = the raster dataset containing the region
         :param sensor_model: Optional[SensorModel] = the sensor model for this raster dataset
 
-        :return None
+        :return: None
         """
         # Set aside the first region
         first_region = all_regions.pop(0)
         for region in all_regions:
-            logger.info(f"Queueing region: {region}")
+            logger.debug(f"Queueing region: {region}")
 
-            region_pixel_bounds = f"{region[0]}{region[1]}"
-            region_id = f"{region_pixel_bounds}-{token_hex(16)}"
             region_request = RegionRequest(
-                image_request.get_shared_values(), region_bounds=region, region_id=region_id, image_extension=image_extension
+                image_request.get_shared_values(),
+                region_bounds=region,
+                region_id=f"{region[0]}{region[1]}-{image_request.job_id}",
```

Comment:
I am trying to decipher this. What if the value of the region_pixel_bounds are the same for other RegionRequest?  region_id would never be unique because of this.

Reply:
The region pixel bounds, on a per image request basis, always have to be unique as they are the unique region coordinates associated with that region?